### PR TITLE
Update repo to support WCSS and tokens MAJOR updates 

### PR DESCRIPTION
**Pull request by blackfalcon**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** #113, #40

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

This commit updates to the new major releases of WCSS and auro design tokens. Due to code changes in Auro design tokens, all references to --auro were updated to --ds. Other token ref changes are due to deprecated tokens.

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]
Need to run a build here. The api.md file should have a change because of the typos you fixed.

**Comment by blackfalcon** [comment]
> Need to run a build here. The api.md file should have a change because of the typos you fixed.

Build was updated and committed. FYI... this code is supposed to run the build in the remote and then commit back to the repo. I created a new issue to look into this. 

- https://github.com/AlaskaAirlines/WC-Generator/issues/426

https://github.com/AlaskaAirlines/auro-checkbox/blob/03777edf3c5e93246f40cc6bd32d20e69b8e628f/package.json#L110-L117

**Comment by blackfalcon** [comment]
This PR is blocked as the dependency on auro icons was reverted 

- https://github.com/AlaskaAirlines/Icons/issues/211

This update will need to be re-applied and a solution shipped for auro-icon shipped as well before this can be completed. 

- https://github.com/AlaskaAirlines/auro-icon/issues/94

**Comment by Patrick-Daly-AA** [comment]
@blackfalcon , "pick this up again and finish it"

File: src/auro-checkbox.js [hunk 1]
```diff
@@ -15,15 +15,15 @@ import checkLg from '@alaskaairux/icons/dist/icons/interface/check-lg_es6.js';
  * The auro-select element is a wrapper for auro-dropdown and auro-menu to create a dropdown menu control.
  *
  * @attr {Boolean} checked - If set to true, the checkbox will be filled with a checkmark.
- * @attr {Boolean} disabled - If set to true, the checkbox will be unclickable.
+ * @attr {Boolean} disabled - If set to true, the checkbox will be not clickable.
```
Comment (jason-capsule42) [inline src/auro-checkbox.js:18]:
```suggestion
 * @attr {Boolean} disabled - If set to true, the checkbox will not be clickable.
```

File: src/auro-checkbox.scss [hunk 2]
```diff
@@ -19,70 +19,21 @@
...
-@import './../node_modules/@aurodesignsystem/webcorestylesheets/dist/utilityClasses/displayProperties';
+@import "./../node_modules/@aurodesignsystem/webcorestylesheets/src/breakpoints";
+@import "./../node_modules/@aurodesignsystem/webcorestylesheets/src/core";
+@import './../node_modules/@aurodesignsystem/webcorestylesheets/src/utilityClasses/displayProperties';
 
 // container styles - defines the look of the outer custom component
 :host {
   display: block;
-  --cbxLocation: var(--auro-size-50);
+  --cbxLocation: var(--ds-size-50);
```
Comment (jason-capsule42) [inline src/auro-checkbox.scss:31]:
```suggestion
  --cbxLocation: var(--ds-size-50, $ds-size-50);
```

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]
Local host of this branch doesn't work with tabbing focus to a checkbox but it does work on the docsite.

Localhost API page examples - checkbox is being rendered on the wrong side of the element.

**Comment by jordanjones243** [comment]
> Local host of this branch doesn't work with tabbing focus to a checkbox but it does work on the docsite.
> 
> Localhost API page examples - checkbox is being rendered on the wrong side of the element.
> 
> Screenshot 2024-01-08 at 7 48 52 PM

This issue does not occur on my localhost, we can discuss this further together if this bug still persists for you. All other issues have been addressed.

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**